tests/e2e/test_token_details.py

```diff
@@ -14,7 +14,7 @@ def test_token_decimals(self):
         # Goerli doesn't seem to have tokens with anything other than 18 decimals.
         cow = "0x3430d04E42a722c5Ae52C5Bffbf1F230C2677600"
         self.assertEqual(get_token_decimals(W3, cow), 18)
-        self.assertEqual(get_token_decimals(W3, duneapi.types.Address(cow)), 18)
+        self.assertEqual(get_token_decimals(W3, dune_client.types.Address(cow)), 18)
```

Comment:
Nitpick suggestion:
```suggestion
        self.assertEqual(get_token_decimals(W3, Address(cow)), 18)
```

Reply:
Right along with the suggestion above this is much better.

Reply:
Fixed.